values.yaml

```diff
@@ -305,3 +305,35 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+
+
+## @section Realm creator parameters
+## @descriptionStart
+## This Helm chart provides the possibility to create a realm after Keycloak was deployed. This is done using [keycloak-config-cli](https://github.com/adorsys/keycloak-config-cli). All options from realmCreator.options are passed to this cli.
+## @descriptionEnd
+
+realmCreator:
```

Comment:
more realms! can we have multiple? :innocent:

Reply:
Yes, we can https://github.com/intensiongmbh/keycloak-helm-chart/pull/3/commits/99340edcdd90efd4303b3170db6676b0d1d38b8b